### PR TITLE
make release 0.10.1

### PR DESCRIPTION
**Pull request by dinhxuanvu**
Signed-off-by: Vu Dinh <vdinh@redhat.com>

### CODE REVIEW

File: manifests/0000_50_olm_12-olm-operators.catalogsource.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
   name: olm-operators
-  namespace: openshift-operator-lifecycle-manager
+  namespace: openshift-marketplace
```
Comment (ecordell) [inline manifests/0000_50_olm_12-olm-operators.catalogsource.yaml]: I don't think this should've changed to openshift-marketplace, I think it should stay in the olm namespace.

This looks like it's causing the e2e failures as well.

Comment (alecmerdler) [reply manifests/0000_50_olm_12-olm-operators.catalogsource.yaml]: That's probably from #871. We might need another variable for that.

Comment (dinhxuanvu) [reply manifests/0000_50_olm_12-olm-operators.catalogsource.yaml]: I will revert that change my PR.

Comment (dinhxuanvu) [reply manifests/0000_50_olm_12-olm-operators.catalogsource.yaml]: On the second thought, there are changes like this across other files so I need to know if we are gonna switch all of them back to olm namespace or just this one specifically.

Comment (ecordell) [reply manifests/0000_50_olm_12-olm-operators.catalogsource.yaml]: it should just be this one that needs to change, from what I can recall

**Pull request merged**